store/tikv/snapshot.go

```diff
@@ -785,5 +834,17 @@ func (rs *SnapshotRuntimeStats) String() string {
 		d := time.Duration(ms) * time.Millisecond
 		buf.WriteString(fmt.Sprintf("%s_backoff:{num:%d, total_time:%s}", k.String(), v, execdetails.FormatDuration(d)))
 	}
+	if rs.scanDetail != nil {
+		buf.WriteString(fmt.Sprintf(", rocksdb{%s: %d, %s: %d, %s: %d, %s: %d, %s: %d, %s: %d, %s: %d, %s: %d, %s: %d}",
+			execdetails.ProcessKeysStr, rs.scanDetail.ProcessedKeys,
+			execdetails.TotalKeysStr, rs.scanDetail.TotalKeys,
+			execdetails.RocksdbDeleteSkippedCountStr, rs.scanDetail.RocksdbDeleteSkippedCount,
+			execdetails.RocksdbKeySkippedCountStr, rs.scanDetail.RocksdbKeySkippedCount,
+			execdetails.RocksdbBlockCacheHitCountStr, rs.scanDetail.RocksdbBlockCacheHitCount,
+			execdetails.RocksdbBlockReadCountStr, rs.scanDetail.RocksdbBlockReadCount,
+			execdetails.RocksdbBlockReadByteStr, rs.scanDetail.RocksdbBlockReadByte,
+			execdetails.ProcessTimeStr, rs.scanDetail.ProcessTime,
+			execdetails.WaitTimeStr, rs.scanDetail.WaitTime))
```

Comment:
```sql
BatchGet:{num_rpc:1, total_time:1.48ms}, rocksdb{Process_keys: 3, Total_keys: 3, Rocksdb_delete_skipped_count: 0, Rocksdb_key_skipped_count: 0, Rocksdb_block_cache_hit_count: 8, Rocksdb_block_read_count: 0, Rocksdb_block_read_byte: 0, Process_time: 0, Wait_time: 0}
```

I think the `Process_time` and  `Wait_time` doesn't belong to `rockdb` runtime information, Do you have any suggestion about this `String` display? @breeswish

Reply:
Yes, they don't belong to RocksDB, and should be displayed separately. It could be:

num_rpc: 1, total_rpc_time: 1.48ms, total_process_time: xxxms, total_wait_time: xxxms